ansible/veos_vtb

```diff
@@ -113,7 +114,7 @@ all:
           type: kvm
           hwsku: msft_multi_asic_vs
           serial_port: 9005
-          num_asics: 6
+          num_asics: 4
```

Comment:
this should be kept.

I am ok to introduce a new one since in the long term we need to run vs test with 6 asics.

Reply:
Added a new hwsku for 4-asics and kept the 6-asic sku as is.